src/js/components/TableCell/doc.js

```diff
@@ -10,8 +10,12 @@ export const doc = TableCell => {
     .intrinsicElement('td');
 
   DocumentedTableCell.propTypes = {
-    plain: PropTypes.bool
-      .description('Whether default styling context should be removed.')
+    plain: PropTypes.oneOfType([PropTypes.bool, PropTypes.oneOf(['noPad'])])
```

Comment:
We should not expose `noPad`

Reply:
We need to leave here for typing to pass, but since it's internal use only we can remove it from site documentation.